src/components/Data/Documents/Page.vue

```diff
@@ -484,6 +484,19 @@ export default {
         this.$log.error(e)
         if (e.status === 412) {
           this.indexOrCollectionNotFound = true
+        } else if (e.message.includes('failed to create query')) {
```

Comment:
```suggestion
        } else if (e.id === 'services.storage.unexpected_bad_request') {
```

Reply:
@jenow not available yet since we're still on the SDK 7.0.0-beta (which is shit, we should bump ASAP).
Once we bumped, your remark must be extended to all the cases where we perform pattern matching on the SDK error messages.